Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -321,23 +314,23 @@
         android:label="SubmissionQRCodeScanFragment">
         <action
             android:id="@+id/action_submissionQRCodeScanFragment_to_submissionDispatcherFragment"
-            app:destination="@id/submissionDispatcherFragment"
-            app:popUpTo="@id/submissionDispatcherFragment"
+            app:popUpTo="@id/submissionQRCodeScanFragment"
             app:popUpToInclusive="true" />
         <action
             android:id="@+id/action_submissionQRCodeScanFragment_to_submissionResultFragment"
             app:destination="@id/submissionResultFragment"
-            app:popUpTo="@id/submissionResultFragment">
+            app:popUpTo="@id/mainFragment"
+            app:popUpToInclusive="false" >
             <argument
                 android:name="skipInitialTestResultRefresh"
                 android:defaultValue="true"
                 app:argType="boolean" />
         </action>
-        <action
+         <action
             android:id="@+id/action_submissionQRCodeScanFragment_to_submissionTestResultAvailableFragment"
             app:destination="@id/submissionTestResultAvailableFragment"
             app:popUpTo="@id/mainFragment"
-            app:popUpToInclusive="true" />
```

Comment:
We need to leave the mainFragment on the stack, so going back brings us there before closing the app